lametro/models.py

```diff
@@ -408,7 +408,10 @@ def upcoming_committee_meetings(cls):
 
     @property
     def media(self):
-        return LAMetroEventMedia.objects.filter(event_id=self.ocd_id)
+        ## order_by makes sure that 'Watch in English' appears first
```

Comment:
this comment would function better as a doc string, like: "Return related LAMetroEventMedia objects such that English audio appears first"